src/main/g8/build.sbt

```diff
@@ -0,0 +1,17 @@
+import ProjectPlugin._
+
+lazy val protocol = project in file("protocol") settings rpcProtocolSettings
```

Comment:
Personally, and based on the current standards in the frees-io projects we are not using the infix syntax to express the sbt settings.

`lazy val protocol = project.in(file("protocol")).settings(rpcProtocolSettings)`